lib/ansible/plugins/lookup/passwordstore.py

```diff
@@ -95,11 +100,11 @@ def parse_params(self, term):
 
     def check_pass(self):
         try:
-            self.passoutput = check_output2(["pass", self.passname]).splitlines()
+            self.passoutput = to_native(check_output2(["pass", self.passname])).splitlines()
```

Comment:
i've changed that line to this:
``` python
            self.passoutput = to_text(check_output2(["pass", self.passname]), errors='surrogate_or_strict').splitlines()
```

and run the integration tests for passwordstore on python-3.5.  It worked.  So I think we can merge this with that change.